include/sfx.h

```diff
@@ -241,6 +240,7 @@ typedef enum { MONO_SOUND, STEREO_SOUND } soundMode;
 
 // STAGE DRE
 #define NA_SE_SU_SHOOT_PINKBALLS 0x62C
+#define SE_SUC_REVEAL 0x637
```

Comment:
Is updating the locations where these are used supposed to be a part of this? For example https://github.com/Xeeynamo/sotn-decomp/blob/140a5e6435cf317914aac89e5114322c806de5d2/src/st/dre/1697C.c#L44

Reply:
I can start going through the codebase and updating those entries.